safor/index.html

```diff
@@ -0,0 +1,125 @@
+        <div class="text">
+                <h1>WHY SAFOR?</h1>
+                <ul>
+                    <li><h1>Reliability</h1><p>You do not need to worry that you will be far from the city or you will be hard to find in a urban jungle</p></li>
+                    <li><h1>Speed</h1><p>Do not waste time explaining the situation and describing what is happening</p></li>
+                    <li><h1>Anonymity</h1><p>Your relatives can be notified that you need help only with your consent</p></li>
+                </ul>
+        </div>
+    </section>
+    <section class="fifth-section">
+        <div class="white-rect">
+            <div class="for-people-container">
+                    <h2>FOR PEOPLE</h2>
+            </div>
+            <ul>
+                <li><img src="images/section5/pentagon.svg" alt="pentagon"></li>
+                <li>Do not waste time to explain your location and nature of the threat, guard patrols get your alarm in 1 second and advances to you no matter whether you are in motion or not.</li>
+                <li><img src="images/section5/star.svg" alt="star"></li>
+                <li>You will be foundregardless of whether you are in town or on a trip. The service covers the entire territory of the country and allows you to specify the exact location even when in a crowded place</li>
+                <li><img src="images/section5/triangle.svg" alt="triangle"></li>
+                <li>And we will help in any situation. Only after coming to you the calling will be closed by the patrol arrived, whose goal is to protect and help you.</li>
+            </ul>
+        </div>
+        <div class="black-rect">
+            <h2>FOR BUSINESS</h2>
+            <ul>
+                <li><img src="images/section5/rectangle.svg" alt="rectangle"></li>
+                <li>At the heart of our activities is an immediate response to incoming proposals from our clients and partners, changing external and internal factors and signals for help.</li>
+                <li><img src="images/section5/circle.svg" alt="circle"></li>
+                <li>Business proposals, external and internal requests and new technologies. Transparency of all processes of interaction with our employees, customers and business partners.</li>
+                <li><img src="images/section5/hexagon.svg" alt="hexagon"></li>
+                <li>In making decisions, the quality of the services provided and the distribution of funds. We use all available resources to achieve maximum results in our work and communication.</li>
+            </ul>
+        </div>
+    </section>
+    <section class="form">
+        <form action="#">
+            <h1>HAVE QUESTIONS? CONTACT US</h1>
+            <div class="first-row">
+                <input type="text" class="name" placeholder="name">
+                <input type="email" class="email" placeholder="email">
+            </div>
+            <input type="text" class="message" placeholder="message">
```

Comment:
As i can see, distance between placeholder and border is different on 375 resolution and less